editor/editor_inspector.cpp

```diff
@@ -1761,6 +1761,10 @@ void EditorInspector::update_tree() {
 			continue;
 		}
 
+		if (p.name == "script") {
+			category_vbox = nullptr; // script should go into its own category
+		}
+
```

Comment:
Doesn't look like related to this PR.

Reply:
It fixes a bug where the script appears above the process mode property, so I fixed it here.